Description: Do not print to stderr in test suite
Author: Petter Reinholdtsen <pere@debian.org>

Forwarded: yes
Status: Applied upstream
Reviewed-By: Petter Reinholdtsen <pere@debian.org>
Last-Update: 2014-04-17

--- startpar-0.59.orig/testsuite/runtests
+++ startpar-0.59/testsuite/runtests
@@ -12,7 +12,7 @@ cat > etc/init.d/test <<EOF
 #!/bin/sh
 set -e
 ### BEGIN INIT INFO
-# Provides:          stop-readahead-fedora
+# Provides:          test
 # Required-Start:    $all
 # Required-Stop:     $remote_fs
 # Default-Start:     2 3 4 5
@@ -25,7 +25,11 @@ EOF
 chmod a+rx etc/init.d/test
 
 /sbin/insserv -p etc/init.d test
-$STARTPAR -d etc/init.d -e etc -P S -R 2 -M start
+if $STARTPAR -d etc/init.d -e etc -P S -R 2 -M start 2>&1 | grep -q 'is running' ; then
+    echo success: the test init.d script was running
+else
+    echo error: the test init.d was not running
+fi
 
 rm -rf etc
 
